### ISSUE TITLE
[RHEL/6] Numerous "xccdf_org.ssgproject.content_rule_partition_for_var_log_audit - At least one <xccdf:check-content-ref> element MUST be provided in each <xccdf:check> " errors reported by the NIST SCAP content validation testsuite

### ISSUE THREAD

**Issue by iankko** [OpenSCAP/scap-security-guide#1038]
Example (https://jlieskov.fedorapeople.org/scap-scapversion-1.2-validation-result.html):
```
175-1 	1
(of 294) 	ERROR 	SCHEMATRON 	xccdf_org.ssgproject.content_rule_partition_for_var_log_audit - At least one <xccdf:check-content-ref> element MUST be provided in each <xccdf:check> 
```

There are 294 instances of the very same error message.

**Comment by shawndwells** [comment]
@iankko  - Since https://github.com/OpenSCAP/scap-security-guide/pull/1050 was merged, can we close this issue?

**Comment by iankko** [comment]
That's correct. This got fixed via #1050 . Closing. Thanks.
